counsel.el

```diff
@@ -730,6 +730,12 @@ a symbol and how to search for them."
   "Face used by `counsel-M-x' for key bindings."
   :group 'ivy-faces)
 
+;;** `counsel-yank-pop'
+(defface counsel-separator
```

Comment:
The docstring sounds pretty generic, perhaps this face should be added straight to `ivy.el`?